[2022-07-05 09:32:58,354] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sql_branch_dag.branch_sql manual__2022-07-05T09:32:57.537159+00:00 [queued]>
[2022-07-05 09:32:58,381] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sql_branch_dag.branch_sql manual__2022-07-05T09:32:57.537159+00:00 [queued]>
[2022-07-05 09:32:58,382] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 09:32:58,386] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-05 09:32:58,387] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 09:32:58,421] {taskinstance.py:1377} INFO - Executing <Task(BranchSQLOperator): branch_sql> on 2022-07-05 09:32:57.537159+00:00
[2022-07-05 09:32:58,441] {standard_task_runner.py:52} INFO - Started process 556 to run task
[2022-07-05 09:32:58,456] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sql_branch_dag', 'branch_sql', 'manual__2022-07-05T09:32:57.537159+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/sql_branch_operator.py', '--cfg-path', '/tmp/tmprqsvo1an', '--error-file', '/tmp/tmp690hv5kw']
[2022-07-05 09:32:58,459] {standard_task_runner.py:80} INFO - Job 129: Subtask branch_sql
[2022-07-05 09:32:58,582] {task_command.py:370} INFO - Running <TaskInstance: sql_branch_dag.branch_sql manual__2022-07-05T09:32:57.537159+00:00 [running]> on host dfb99dfc9d64
[2022-07-05 09:32:58,797] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sql_branch_dag
AIRFLOW_CTX_TASK_ID=branch_sql
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T09:32:57.537159+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-05T09:32:57.537159+00:00
[2022-07-05 09:32:58,800] {sql.py:513} INFO - Executing: SELECT count(1) FROM user (with parameters None) with connection: postgres_con
[2022-07-05 09:32:58,825] {base.py:68} INFO - Using connection ID 'postgres_con' for task execution.
[2022-07-05 09:32:59,075] {base.py:68} INFO - Using connection ID 'postgres_con' for task execution.
[2022-07-05 09:32:59,087] {sql.py:531} INFO - Query returns 1, type '<class 'int'>'
[2022-07-05 09:32:59,090] {skipmixin.py:140} INFO - Following branch user_exist
[2022-07-05 09:32:59,091] {skipmixin.py:175} INFO - Skipping tasks ['user_not_exist']
[2022-07-05 09:32:59,172] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=sql_branch_dag, task_id=branch_sql, execution_date=20220705T093257, start_date=20220705T093258, end_date=20220705T093259
[2022-07-05 09:32:59,221] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-05 09:32:59,374] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
